plugins/janus_videoroom.c

```diff
@@ -1007,15 +1100,116 @@ room-<unique room ID>: {
 \endverbatim
  *
  * Once this is done, all that's needed is waiting for the WebRTC PeerConnection
- * establishment to succeed. As soon as that happens, the Streaming plugin
- * can start relaying media from the mountpoint the viewer subscribed to
- * to the viewer themselves.
+ * establishment to succeed. As soon as that happens, the VideoRoom plugin
+ * can start relaying media the recipient subscribed to.
+ *
+ * Once a WebRTC PeerConnection has been established for a subscriber, in
+ * case you want to update a subscription you have to use the \c subscribe
+ * and \c unsubscribe methods: as the names of the requests suggest, the
+ * former allows you to add more streams to subscribe to, while the latter
+ * instructs the plugin to remove streams you're currently subscribe to.
+ * Both requests will trigger a renegotiation, if they were successful,
+ * meaning the plugin will send you a new JSEP offer you'll have to reply
+ * to with an answer: to send the answer, just use the same \c start request
+ * we already described above. Notice that renegotiations may not be
+ * triggered right away, e.g., whenever you're trying to update a session
+ * and the plugin is still in the process of renegoting a previous update
+ * for the same subscription: in that case, an update will be scheduled
+ * and a renegotiation will be triggered as soon as it's viable.
+ *
+ * The syntax of the \c subscribe mirrors the one for new subscriptions,
+ * meaning you use the same \c streams array to address the new streams
+ * you want to receive, and formatted the same way:
+ *
+\verbatim
+{
+	"request" : "subscribe",
+	"streams" : [
+		{
+			"feed_id" : <unique ID of publisher owning the new stream to subscribe to>,
```

Comment:
Same as above, this is also just `feed`.

Reply:
Thanks, fixed both (actually three, "unsubscribe" had the same typo).